lib/rules/color-no-hex/index.js

```diff
@@ -42,7 +42,8 @@ const rule = (primary) => {
 				const endIndex = index + node.value.length;
 
 				report({
-					message: messages.rejected(node.value),
+					message: messages.rejected,
+					messageArgs: [node.value],
```

Comment:
remark

If I understand correctly, the order will be arbitrary.
e.g. if you don't use `--fix` but you want to use the fixed _value_ in the message

i.e.
- we would generate the value but not replace with it in the file
- the user will need to know that the bad value is the first element and the good value is the second element

Reply:
@Mouvedia Thanks for the share. Other problems might be found than you pointed out, but this feature is still experimental. We would try considering it based on reactions or feedback later.